### ISSUE TITLE
BlzGetLocalSpecialEffectZ - is the value with respect to terrain height?

### ISSUE THREAD

**Issue by Cokemonkey11** [lep/jassdoc#30]

**Comment by WaterKnight** [comment]
No, it is the absolute world coordinate but 0 when target-attached. Non-target-attached effects start with the terrain height, however (and they are not clipped, i.e., changing the terrain height does not affect the height of the effects on it).
